lib/sedna/backend/tensorflow/__init__.py

```diff
@@ -30,6 +31,19 @@
     Session = tf.Session
 
 
+_url = os.getenv("S3_ENDPOINT_URL", "http://s3.amazonaws.com")
+if not (_url.startswith("http://") or _url.startswith("https://")):
+    _url = f"https://{_url}"
+url = urlparse(_url)
+use_ssl = url.scheme == 'https' if url.scheme else True
+
+
+os.environ["AWS_ACCESS_KEY_ID"] = os.getenv("ACCESS_KEY_ID", "")
+os.environ["AWS_SECRET_ACCESS_KEY"] = os.getenv("SECRET_ACCESS_KEY", "")
+os.environ["S3_ENDPOINT"] = url.netloc
```

Comment:
Why don't use `ACCESS_KEY_ID` and `ACCESS_KEY_ID`

Reply:
This is the specified key in the framework(tensorflow)